pages/upgrade.rst

```diff
@@ -18,4 +18,9 @@ You should always follow minor versions when updating across multiple versions t
 The Virtual Machine Appliance (OVA) and Amazon Web Services (AMI) Version of Graylog use the Omnibus package. The upgrade documentation is :ref:`part of the graylog-ctl documentation <upgrade_graylog_omnibus>`.
 
 Should the current installation use operating system packages, update the repository package to the target version and use the system tools to upgrade the package.
-For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yum-dnf>` and for .deb based systems :ref:`this update guide <operating_package_upgrade_DEB-APT>` should help.
\ No newline at end of file
+For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yum-dnf>` and for .deb based systems :ref:`this update guide <operating_package_upgrade_DEB-APT>` should help.
+
+Upgrading Elasticsearch
+=======================
+
+When you update the Elasticsearch Cluster from 2.x to 5.x be sure to read `the upgrade guide <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/setup-upgrade.html>`_ provided by elastic. The Graylog :ref:`Elasticsearch configuration documentation <configuring_es>` contains information about the compatible Elasticsearch version. 
```

Comment:
We should add a hint or note here which informs the user to rotate indices when having upgraded from Elasticsearch 2.x to 5.x.